interpret_date.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
+  spec.add_development_dependency "activerecord", ">=4.2"
```

Comment:
Should this be a [runtime dependency](http://guides.rubygems.org/patterns/#runtime-vs-development)?